src/Mollie/WC/Gateway/Abstract.php

```diff
@@ -1552,7 +1555,7 @@ public function onOrderReceivedTitle( $title, $id = null ) {
 			}
 
 			// Checks and title for pending/open orders
-			if ( version_compare( WC_VERSION, '3.0', '<' ) ) {
+			if ( version_compare( mollieWooCommerceWcVersion(), '3.0', '<' ) ) {
 				$payment = Mollie_WC_Plugin::getPaymentObject()->getActiveMolliePayment( $order->id );
 			} else {
 				$payment = Mollie_WC_Plugin::getPaymentObject()->getActiveMolliePayment( $order->get_id() );
```

Comment:
I have a feeling like this project will lose 20% of its code if someday we well refactor places like this like

```php
$orderId = version_compare( mollieWooCommerceWcVersion(), '3.0', '<' ) ? $order->id : $order->get_id();
$payment = Mollie_WC_Plugin::getPaymentObject()->getActiveMolliePayment($orderId);
```

But I know, it's out of this PR's scope and it's too much work to do it just 'by the way'. Also, I guess we will bring `WC` < `3.0` support before such refactoring may happen.